source/triggers/database-triggers.txt

```diff
@@ -681,3 +681,32 @@ limit the number of fields that the Trigger processes by using a
       }
 
 .. include:: /includes/triggers-examples-github.rst
+
+.. _db-triggers-troubleshooting:
+
+Troubleshooting
+---------------
+
+Change Stream History Lost
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When a Database Trigger tries to process an event that has fallen off the
+:manual:`oplog </core/replica-set-oplog/>`, the Trigger becomes
+suspended. When a ``ChangeStreamHistoryLost`` error
+occurs, the following description appears in the App logs:
+
+.. code-block::
+   :copyable: false
+
+   (ChangeStreamHistoryLost) Resume of change stream was not possible,
+   as the resume point may no longer be in the oplog.
+
+You can :manual:`restart </changeStreams/#resume-tokens>` the Trigger, but
+the Trigger will not process any events between the last processed event
+and the time that you restart the Trigger.
+
+.. seealso::
+
+   - For more information, refer to :ref:`Suspended Triggers <suspended_triggers>`.
+   - If you frequently see this error, consider
+     :ref:`Disabling event ordering <database-triggers-disable-event-ordering>`.
```

Comment:
now that i see this content in the context of this page, i think that we don't need this as a dedicated section at all. it's very similar to the content in the [Suspended Triggers section](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/ia-reference/triggers/database-triggers/#suspended-triggers). 

haven't done a super close read, but i think that there isn't any info in here that's not more thoroughly covered in the Suspended Triggers section.

Reply:
Yeah fair point. I'm all for deleting this entirely!